scraper/items.py

```diff
@@ -0,0 +1,49 @@
+from dataclasses import dataclass, field
+from datetime import datetime
+from enum import Enum
+from typing import Optional, Sequence
+
+"""
+These just replicate the database specification. See the backend folder for details.
+"""
+
+
+@dataclass
+class Motion:
+    motie_id: int
+    title: str
+    submit_date: datetime
+    vote_date: datetime
+    document_url: str
+    total_votes: int
```

Comment:
Dit repliceert de backend implementatie, hier zullen vast nog wat aanpassingen aan nodig zijn aangezien ik bijvoorbeeld de submitter ID moet gaan vinden op basis van de persoonsnaam. Dat soort handelingen kunnen we mss beter in de database pipeline zetten.